docs/windows.md

```diff
@@ -1,142 +1,141 @@
-go get -u github.com/whyrusleeping/gx-go
-cd %GOPATH%/src/github.com/ipfs/go-ipfs
-gx --verbose install --global
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
+```
+This can be achieved with the `setx` command:
+```
+SETX PATH %PATH%;\msys64\usr\bin
+```
+
+## Cygwin
+1. Install Cygwin (https://www.cygwin.com)  
+2. During the install, select the following packages. (If you already have Cygwin installed, run the setup file again to install additional packages.) A fresh install should look something like [this reference image](https://ipfs.io/ipfs/QmaYFSQa4iHDafcebiKjm1WwuKhosoXr45HPpfaeMbCRpb/cygwin%20-%20install.png).
+3. 
+    - devel packages
+        - `git`
+        - `make`
+    - archive packages
+        - `unzip`
+    - net packages
+        - `curl`  
+4. Run the following inside a normal `cmd` prompt (Not the Cygwin prompt, we only need Cygwin's tools)  
+An explanation of this block is below.
+```
+SET PATH=%PATH%;\cygwin64\bin
+mkdir %GOPATH%\src\github.com\ipfs
+cd %GOPATH%\src\github.com\ipfs
+git clone https://github.com/ipfs/go-ipfs.git
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
+%GOPATH%\bin\ipfs.exe version --all
 ```
 
-## Manual Way: build go-ipfs
+If there were no errors, the final command should output version information similar  to "`ipfs version 0.4.14-dev-XXXXXXX`" where "XXXXXXX" should match the current short-hash of the `go-ipfs` repo. You can retrieve said hash via this command: `git rev-parse --short HEAD`. If `ipfs` runs and the version string matches, building was successful.
 
-To actually build go-ipfs, first go to the cmd/ipfs directory:
+|Command|Explanation|
+| ---: | :--- |
+|`SET PATH=%PATH%;\cygwin64\bin`           |Add Cygwin's bin to PATH; Defaults to: (\cygwin64\bin)|
+|`mkdir %GOPATH%\src\github.com\ipfs`<br/>`cd %GOPATH%\src\github.com\ipfs`<br/>`git clone https://github.com/ipfs/go-ipfs.git`       |Fetch / Update `go-ipfs` source|
+|`cd %GOPATH%\src\github.com\ipfs\go-ipfs` |Change to `go-ipfs` directory|
+|`make install`                            |Build and install to `%GOPATH%\bin\ipfs.exe`|
+|`%GOPATH%\bin\ipfs.exe version --all`     |Test the built binary|
 
+**Tip**: You can permanently append the Cygwin bin to your `PATH` so that on subsequent builds you can simply run
 ```
-cd cmd\ipfs
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
 ```
-
-Then get the current Git commit:
-
+This can be achieved with the `setx` command: 
 ```
-git rev-parse --short HEAD
+SETX PATH %PATH%;\cygwin64\bin
 ```
 
-It will output a small number of hex characters that you must pass to
-the actual build command (replace XXXXXXX with these characters):
+## Minimal
+While it's possible to build `go-ipfs` with `go` alone, we'll be using `git` and `gx` for practical source management.  
+You can use whichever version of `git` you wish but the Windows builds at <https://git-scm.com> are recommended. `git` must be in your `PATH` for `go get` to recognize and use it.
```

Comment:
Same note about passive active voice here as in the intro: “…but **we recommend** the Windows builds at <https://git-scm.com>.” (And if this recommendation is from someone/somewhere else, say their name instead of “we.”)